src/content/structured/components/back-to-top/guidance.mdx

```diff
@@ -81,6 +103,15 @@ Only use the provided back to top component. Don't use any other button in place
   caption="Don’t use any other component for providing back to top functionality. Use the back to top button."
 />
 
+Avoid using the default back to top button when zoomed, as it may obscure important page content. The icon back to top button should be used instead.
```

Comment:
```suggestion
Avoid using the default back to top button when zoomed, as it may obscure important page content. The icon-only back to top button should be used instead.
```